src/components/Boilerplate.tsx

```diff
@@ -47,7 +47,7 @@ const Wrapper = styled.div`
     }
 `
 
-export const BoilerPlate = (p: {title: string, maxHeight?: number, children: any}) => <IndexLayout title={p.title}>
+export const BoilerPlate = (p: {title: string, maxHeight?: number, children: any, canonical: string}) => <IndexLayout title={p.title} >
```

Comment:
Don't you also need to pass this `canonical` attribute to `IndexLayout`?

```suggestion
export const BoilerPlate = (p: {title: string, maxHeight?: number, children: any, canonical: string}) => <IndexLayout title={p.title} canonical>
```

Reply:
Yes I have to. I didn't realized.